### PR TITLE
Fixed Diode Clipper Image

### PR DESCRIPTION
**Pull request by paltrickontpb**
Part of Issue #206 

The previous version did not have diode clipper circuit diagrams:

2018-10-15-230357_1024x768_scrot
2018-10-15-230313_1024x768_scrot

Circuits have been added to fix this error. The output is tested and shown below:

2018-10-15-231219_1024x768_scrot
2018-10-15-231222_1024x768_scrot

### CODE REVIEW

**Comment by paltrickontpb** [comment]
Just to clarify, are these alright? If so, then i will be making these changes ASAP.

batt
diode

**Comment by paltrickontpb** [comment]
Here are the changes made,

Series and shunt clipper circuits



I also added the Clamper circuit schematic


Please do check it out @harsh-2711

**Comment by paltrickontpb** [comment]
@CloudyPadmal I have updated the repo with svg files and squashed all the commits. Please do check it out whenever you can. Thank You!

**Comment by CloudyPadmal** [comment]
@paltrickontpb I have a question. `PV3` is a pin. Can you point out to how and where I can connect that single pin in the given schematics? It should be a label box same as `GND`, `CH1` etc.
Also can you provide the source you got the circuit diagrams from? Thanks.

File: psl_res/HTML/L_DiodeClipping.html [hunk 1]
```diff
@@ -6,16 +6,20 @@
...
 	CH2 monitors the clipped waveform .<br><br>
 	Change the value of PV3, and observe its effect on the output.<br>
 
 	<h3>Series Clipping</h3>
-	<img src="images/diode_clipping_series.svg" width="100%"><br>
+	<img src="images/diodeSeriesClipper.svg" width="100%"><br>
+
+	Make the Connections as shown in the figure.<br>
+	CH1 monitors the original waveform output from W1.<br>
+	CH2 monitors the clipped waveform .<br><br>
```
Comment (CloudyPadmal) [inline psl_res/HTML/L_DiodeClipping.html]: There is an extra space here

File: psl_res/HTML/L_DiodeClipping.html [hunk 2]
```diff
@@ -6,16 +6,20 @@
...
+	Change the value of PV3, and observe its effect on the output.<br>
```
Comment (CloudyPadmal) [inline psl_res/HTML/L_DiodeClipping.html]: No need of a comma after PV3

File: psl_res/HTML/L_DiodeClipping.html (hunk 1 above)
Comment (paltrickontpb) [reply psl_res/HTML/L_DiodeClipping.html]: Hi, i did not understand what you meant by the extra space Could you please refer to the line number. Thank You !

Comment (paltrickontpb) [reply psl_res/HTML/L_DiodeClipping.html]: Got it, It fine,

**Comment by paltrickontpb** [comment]
Will do a PR again due to rebasing errors

**Pull request closed without merging**